conda_smithy/lint_recipe.py

```diff
@@ -660,7 +660,9 @@ def lintify(meta, recipe_dir=None, conda_forge=False):
         expected_exceptions = set([l.strip() for l in expected_exceptions])
     if set(filtered_licenses) - expected_licenses:
         hints.append(
-            "License is not an SPDX identifier (or a custom LicenseRef) nor an SPDX license expression."
+            "License is not an SPDX identifier (or a custom LicenseRef) nor an SPDX license expression.\n\n"
+            "A full list of acceptable licenses can be found "
```

Comment:
The link doesn't give a full list, but a few examples and links to the full list.
```suggestion
            "Documentation can be found "
```

Reply:
Good point. I made a similar change below. Please take a look.